test/e2e/tests/validator_test.go

```diff
@@ -64,6 +64,10 @@ func TestValidator_Propose(t *testing.T) {
 		if node.Mode != e2e.ModeValidator {
 			return
 		}
+		if node.ClockSkew != 0 && node.Testnet.PbtsEnableHeight != 0 {
```

Comment:
This test has a really bad name, as it does not test a validator proposing, but checks how many of those proposals were decided.

Reply:
What do you suggest?

Reply:
Not sure. `TestValidator_ProposalsAccepted` maybe?
Given that the test does not really know what blockchain/manifest will run, it cannot know for how long the test will run, which weights validators have and what is a reasonable number of proposals to be accepted, it is potentially flaky by construction.

Reply:
It does know the weights (it's built into the `validatorSchedule`, check line 72 below). But you're right in that there's no way the test case can know in which round a node decided for every height. But, I'd leave it as it hasn't been flaky for the last months/years.